po/fr.po

```diff
@@ -0,0 +1,545 @@
+msgid "Network"
+msgstr "Réseau sans fil"
+
+#: src/models/shared.js:45
+msgid "Inter-process communications"
+msgstr "Communication inter-processus"
+
+#: src/models/shared.js:78
+msgid "List of subsystems shared with the host system"
+msgstr "Liste des sous-systèmes partagés avec le système hôte"
+
+#: src/models/sockets.js:37
+msgid "X11 windowing system"
+msgstr "Système de fenêtrage X11"
+
+#: src/models/sockets.js:44
+msgid "Wayland windowing system"
+msgstr "Système de fenêtrage Wayland"
+
+#: src/models/sockets.js:51
+msgid "Fallback to X11 windowing system"
+msgstr "Retour au système de fenêtrage X11"
+
+#: src/models/sockets.js:58
+msgid "PulseAudio sound server"
+msgstr "Serveur de son PulseAudio"
+
+#: src/models/sockets.js:65
+msgid "D-Bus session bus"
+msgstr "Bus de session D-Bus"
+
+#: src/models/sockets.js:72
+msgid "D-Bus system bus"
+msgstr "Bus du système D-Bus"
+
+#: src/models/sockets.js:79
+msgid "Secure Shell agent"
+msgstr "Agent Secure Shell"
+
+#: src/models/sockets.js:86
+msgid "Smart cards"
+msgstr "Carte à puce"
+
+#: src/models/sockets.js:93
+msgid "Printing system"
+msgstr "Système imprimante"
+
+#: src/models/sockets.js:118
+msgid "List of well-known sockets available in the sandbox"
+msgstr "Liste des prises connues disponible dans le sac de sable"
+
+#: src/models/variables.js:38
+msgid "Variables"
+msgstr "Variables"
+
+#: src/models/variables.js:41
+msgid "e.g. GTK_DEBUG=interactive"
+msgstr "e.g. GTK_DEBUG=interactive"
+
+#: src/models/variables.js:71
+msgid "List of variables exported to the application"
+msgstr "Liste de variables exportées à l'application"
+
+#: src/widgets/aboutDialog.js:38
+msgid "Contributions by"
+msgstr "Contributions de"
+
+#. TRANSLATORS: <full-month-name> <day-of-month>, <year-with-century>
+#: src/widgets/appInfoViewer.js:61
+msgid "%B %e, %Y"
+msgstr "%B %e, %Y"
+
+#: src/widgets/appInfoViewer.ui:127
+msgid "Version"
+msgstr "Version"
+
+#: src/widgets/appInfoViewer.ui:142
+msgid "Last Updated"
+msgstr "Dernière mise à jour"
+
+#: src/widgets/appInfoViewer.ui:157
+msgid "Runtime"
+msgstr "Runtime"
+
+#: src/widgets/busNameRow.js:33 src/widgets/pathRow.js:71
+#: src/widgets/relativePathRow.js:32 src/widgets/variableRow.js:30
+msgid "This is not a valid option"
+msgstr "Ceci n'est pas une option valide"
+
+#: src/widgets/detailsButton.js:59
+msgid "_Show Details"
+msgstr "_Voir les Détails"
+
+#: src/widgets/detailsButton.js:107
+msgid "Show application in a software manager"
+msgstr "Montrer l'application dans un gestrionnaire de logiciels"
+
+#: src/widgets/detailsButton.js:109
+msgid "No software manager found"
+msgstr "Aucun qestionnaire de logiciel trouvé"
+
+#: src/widgets/menu.ui:7
+msgid "_Help"
+msgstr "_Aide"
+
+#: src/widgets/menu.ui:11
+msgid "_Documentation"
+msgstr "_Documentation"
+
+#: src/widgets/menu.ui:15
+msgid "_Keyboard Shortcuts"
+msgstr "_Raccourcis Claviers"
+
+#: src/widgets/menu.ui:19
+msgid "_About Flatseal"
+msgstr "À propos de Flatseal"
```

Comment:
You're missing the  mnemonic `_` mark, e.g. it could be `À propos de _Flatseal`, but it should be different for each marked string.